go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/traefik/plugindemo
 
-go 1.19
+go 1.22.0
```

Comment:
Do you want to specify patch version or just use `go 1.22` ?

Reply:
I want to use a Go version, not a language version, so yes the `.0` is required.

Reply:
Sounds good!